Extensions/PathfindingBehavior/pathfindingobstacleruntimebehavior.ts

```diff
@@ -105,6 +114,21 @@ namespace gdjs {
         });
       }
     }
+    /**
+     * Tool class with a getAABB method.
+     * @ignore
+     */
+    export class AABBContainer {
```

Comment:
nitpicking: maybe not export the class if we don't use it outside. Otherwise we risk polluting gdjs namespace.